docs/stage/download-raw-data.md

```diff
@@ -10,6 +10,9 @@ To run this:
 
 `$ node ./src/bin/download-raw.js`
 
+It can be stopped at any time and it will not leave the database in a bad state or lose too much work.
```

Comment:
What does "too much" mean? Would it be true to say "only records being processed at that particular moment in time would be lost"?